code/game/objects/items/weapons/handcuffs.dm

```diff
@@ -266,3 +280,20 @@
 
 /obj/item/weapon/restraints/legcuffs/beartrap/energy/attack_hand(mob/user)
 	Crossed(user) //honk
+
+/obj/item/weapon/restraints/legcuffs/bola
+	name = "bola"
+	desc = "A restraining device designed to be thrown at the target. Upon connecting with said target, it will wrap around their legs, making it difficult for them to move quickly."
+	icon_state = "bola"
+	breakouttime = 35//easy to apply, easy to break out of
+
+/obj/item/weapon/restraints/legcuffs/bola/throw_impact(atom/hit_atom)
+	if(..() || !iscarbon(hit_atom))//if it gets caught or the target can't be cuffed,
+		return//abort
+	var/mob/living/carbon/C = hit_atom
+	if(!C.legcuffed)
+		C.legcuffed = src
+		src.loc = C
+		C.update_inv_legcuffed()
+		feedback_add_details("handcuffs","B")

```

Comment:
Keep in mind, on Paradise, these are *not* something that players can make and throw at people--these are a mech weapon that launches them at players---so it's a bit different; strongly agree that these shoudl *not* weaken if players can make them.